app/lib/core/services/database/database_service.dart

```diff
@@ -1,21 +1,17 @@
 import 'dart:async';
 
 import 'package:app/core/models/animal_transport_record.dart';
-import 'package:app/core/models/test_animal_transport_record.dart';
+import 'package:app/test/test_animal_transport_record.dart';
 
 class DatabaseService {
   // TODO: #130. Replace with actual DB functions
   Stream<AnimalTransportRecord> getUpdatedCompleteATRs() async* {
-    while (true) {
```

Comment:
God rid of this because each push of a new record caused excessive rebuilds when playing around on the app.
We really only need the one push. If you sign out and sign back in, the service is remade and will push out a test record after the time completes.